### ISSUE TITLE
sycl-ls for gpus fails on oneapi-basekit 20.04

### ISSUE THREAD

**Issue by tkatila** [intel/oneapi-containers#66]
`sycl-ls` output is different when using 20.04 or 22.04 variant of the oneapi-basekit container:
```
$ sudo docker run -v /dev/dri/by-path:/dev/dri/by-path:rw --device /dev/dri --rm -it intel/oneapi-basekit:2024.0.1-devel-ubuntu20.04 sycl-ls
[opencl:acc:0] Intel(R) FPGA Emulation Platform for OpenCL(TM), Intel(R) FPGA Emulation Device OpenCL 1.2  [2023.16.12.0.12_195853.xmain-hotfix]
[opencl:cpu:1] Intel(R) OpenCL, Intel(R) Xeon(R) Platinum 8480+ OpenCL 3.0 (Build 0) [2023.16.12.0.12_195853.xmain-hotfix]

$ sudo docker run -v /dev/dri/by-path:/dev/dri/by-path:rw --device /dev/dri --rm -it intel/oneapi-basekit:2024.0.1-devel-ubuntu22.04 sycl-ls
[opencl:acc:0] Intel(R) FPGA Emulation Platform for OpenCL(TM), Intel(R) FPGA Emulation Device OpenCL 1.2  [2023.16.12.0.12_195853.xmain-hotfix]
[opencl:cpu:1] Intel(R) OpenCL, Intel(R) Xeon(R) Platinum 8480+ OpenCL 3.0 (Build 0) [2023.16.12.0.12_195853.xmain-hotfix]
[opencl:gpu:2] Intel(R) OpenCL Graphics, Intel(R) Data Center GPU Max 1100 OpenCL 3.0 NEO  [23.35.27191.42]
[opencl:gpu:3] Intel(R) OpenCL Graphics, Intel(R) Data Center GPU Max 1100 OpenCL 3.0 NEO  [23.35.27191.42]
[opencl:gpu:4] Intel(R) OpenCL Graphics, Intel(R) Data Center GPU Max 1100 OpenCL 3.0 NEO  [23.35.27191.42]
[opencl:gpu:5] Intel(R) OpenCL Graphics, Intel(R) Data Center GPU Max 1100 OpenCL 3.0 NEO  [23.35.27191.42]
[ext_oneapi_level_zero:gpu:0] Intel(R) Level-Zero, Intel(R) Data Center GPU Max 1100 1.3 [1.3.27191]
[ext_oneapi_level_zero:gpu:1] Intel(R) Level-Zero, Intel(R) Data Center GPU Max 1100 1.3 [1.3.27191]
[ext_oneapi_level_zero:gpu:2] Intel(R) Level-Zero, Intel(R) Data Center GPU Max 1100 1.3 [1.3.27191]
[ext_oneapi_level_zero:gpu:3] Intel(R) Level-Zero, Intel(R) Data Center GPU Max 1100 1.3 [1.3.27191]
```

This could mean that GPU use with sycl is not possible?

Noticed here: https://github.com/sylabs/singularity/issues/1094

**Comment by tingleby** [comment]
Unfortunately this is expected. The gpu drivers packages are no longer available from Intel for Ubuntu 20.04.

**Comment by tkatila** [comment]
Thanks @tingleby! Would it possible to add that into the deprecation notes? https://www.intel.com/content/www/us/en/developer/articles/release-notes/intel-oneapi-toolkit-release-notes.html
